components/unique/Gallery/Item/AvailableActions.vue

```diff
@@ -311,3 +312,11 @@ export default class AvailableActions extends mixins(RmrkVersionMixin) {
   }
 }
 </script>
+
+<style scoped lang="scss">
```

Comment:
```suggestion
<style scoped>
```
because it does not need `scss`